app/Http/Transformers/EventTransformer.php

```diff
@@ -29,8 +29,8 @@ public function transform(Event $event)
             foreach ($array as $key => $value) {
                 $keyValues = explode(':', $value);
 
-                $key = trim($keyValues[0], '"');
-                $value = trim($keyValues[1], '"');
+                $key = isset($keyValues[0]) ? trim($keyValues[0], '"') : '';
+                $value = isset($keyValues[1]) ? trim($keyValues[1], '"') : '';
                 $content[$key] = $value;
```

Comment:
@chloealee does this error out when `$key` is `''`? can that be an array key?

Reply:
@katiecrane great q! So this line is basically just getting rid of double `""`. Some values were being returned as `"59cff102a0bfad0e3c322e57""` so the `trim` gets rid of the extra `"`. However, if it isn't set, it throws the error `(1/1) ErrorException
Undefined offset: 1`

Reply:
@chloealee right! what I'm wondering is in the case where `$keyValues[0]` isn't set, then `$key` gets set as `''` right? I'm wondering what happens when you do `$content[$key] = $value;` with `$key` set as `''` and if that works because it seems a little funky to me to have `''` as an array key

Reply:
@katiecrane ah I see what you mean now! Makes total sense! So in this case, I think the key will always be set so I removed the `isset` check for the key and just kept on `$value` - thanks for flagging that!

Reply:
@chloealee woo! sorry to confuse you at first!

Reply:
not at all, @katiecrane ! I just wasn't reading carefully! 👓